### ISSUE TITLE
Subs override account names

### ISSUE THREAD

**Issue by chevdor** [polkadot-js/apps#8327]
- [ ] Bug report
  - [x] Feature request
  - [ ] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**

Currently, setting onchain identity or setting an account as a sub overrides the account's name given by the user locally.
This is an issue as you may not want to disclose all account data on chain.

For instance, say you create 2 proxies P1 and P2 for account A. Account A has a verifiied identity. P1 and P2 do not have an identity. P1 and P2 are set has Proxy (say Governance for the sake of the discussion). Locally the user may call P1 and P2 "My account foo on wallet bar". Adding P1 and P2 as sub of A does make sense in order to benefit from the idenity verification of A.
While setting the subs however, the user may not want to be as specific as it was done locally and simply call the proxies: "Gov 1" and "Gov 2". Doing so, the UI renames the accounts to "Gov 1" and "Gov 2", removing the ability for the user to keep seeing the more descriptive local names.

* **What is the motivation for changing the behavior?**

Better separation between onchain data and local data.

* **Please tell us about your environment:**

n/a

* **suggestion**

This could be solved for instance by still showing the local name of the account, in addition to the sub name that replaced it.

**Comment by jacogr** [comment]
Funnily enough, the local names _used_ to override on-chain names. Then we had complaints about "I set my name, but it is not showing, I would like the on-chain names". Generally I also prefer the on-chain names, aka as I set mine to be visible to others.

The only way around it would be to show both, somehow. Not that we have any space.

**Comment by chevdor** [comment]
I think showing both (even if that means showing one smaller) is the way to go as users may alway choose to have more details exposed in the local names.

For instance, your `Jaco` account as defined onchain maybe called `Jaco Blue Ledger/0/12` or `My stash 3` locally and this local naming can be useful locally (if shown as well).

I actually see a significant amount of users using `My Stash` (and vairations...) as display name in their identity. This is legit if you consider that users set the name they want to see locally.

So in short, local names and public/onchain names should be 2 different things. Some accounts will have local only, or onchain only (is that case even possible ?), or both. 

I don't think having an option to let one name override the other is a good option.

There is also the option to only show this in the side panel but ideal, the "other" name should also be searchable.
